model/user.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const crypto = require('crypto');
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+const mongoose = require('mongoose');
+const createError = require('http-errors');
+const Promise = require('bluebird');
+const debug = require('debug')('parkify:user');
+
+const Schema = mongoose.Schema;
+
+const userSchema = Schema({
+  username: { type: String, required: true, unique: true },
```

Comment:
I know this is how we've done this in the past, but I think we should just make this property "name" since user is the model. That way when we access the name of a model instance, we use `user.name` instead of `user.username`.